src/Provider/Provider.Web/Controllers/VacancyPreviewController.cs

```diff
@@ -86,7 +88,32 @@ private void SetSectionStates(VacancyPreviewViewModel viewModel)
             viewModel.TrainingSectionState = GetSectionState(viewModel, new[] { FieldIdentifiers.Training }, true, vm => vm.TrainingType, vm => vm.TrainingTitle);
             viewModel.DisabilityConfidentSectionState = GetSectionState(viewModel, new[]{ FieldIdentifiers.DisabilityConfident}, true, vm => vm.IsDisabilityConfident);
         }
-        
+
+        private int GetSectionStateCount(VacancyPreviewViewModel viewModel)
+        {
+            var count = 0;
+            if (CheckIfSectionIsIncomplete(viewModel.ShortDescriptionSectionState))
+                count++;
+            if (CheckIfSectionIsIncomplete(viewModel.SkillsSectionState))
+                count++;
+            if (CheckIfSectionIsIncomplete(viewModel.DescriptionsSectionState))
+                count += 3;
```

Comment:
Does this need to be incremented by 3? The descriptions all come under a single section